modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -252,12 +286,69 @@ function AddFDSFilterModalContent({
 					value: String(item.id),
 				}));
 
+	const inUseFields = fields.map((item) =>
+		fieldNames?.includes(item.name) ? item.name : undefined
+	);
+
+	const CellRendererDropdown = ({
+		cellRenderers,
+		inUseFields,
+		namespace,
+		onItemClick,
+	}: {
+		cellRenderers: IField[];
+		inUseFields: (string | undefined)[];
+		namespace: string;
+		onItemClick: Function;
+	}) => {
+		return (
+			<ClayDropDown
```

Comment:
This kinda seems odd to be a dropdown with an input as a trigger. Wouldn't it make more sense to be a Select? OR is it ever expected that you would type a value into the trigger?

Reply:
You might check with design on this, as I think it would be more like https://storybook.clayui.com/?path=/story/design-system-components-select--high-level

Reply:
Does Select allow to use custom option objects (for the "IN USE" label)? This is the main reason we choose the actual components

Reply:
Hmm, it may not allow for a label. This is probably an area we need to push back on design or with the clay team, because if design is asking for one thing but we don't have a typical way to do that in Clay, then we should use the clay component and then request that Clay implements the new design in the library.